Anagram/Anagram.cs

```diff
@@ -7,10 +7,36 @@ public class AnagramSelector
     {
         public bool WordPairIsAnagram(string word1, string word2) {
             //Insert the correct implementation here
-            return false;
+            char[] char1 = word1.ToLower().ToCharArray();
```

Comment:
This function may not work for some of the examples in Wikipedia. Make tests for the first three examples in there, to explore the technique of making a failing test first and then fixing it.